pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md

```diff
@@ -45,18 +45,17 @@ kubectl label secret $CLUSTER_NAME-ssh-key clusterctl.cluster.x-k8s.io/move=
 
 ## Create overrides
 
-If your pre-provisioned machines have [overrides](../../../image-builder/override-files) you must create a secret that includes all of the overrides you wish to provide in one file. For example, if you wish to provide an override with Docker credentials and a different source for EPEL on a CentOS7 machine, you can create a file like this:
+If your pre-provisioned machines have [overrides](../../../image-builder/override-files) you must create a secret that includes all of the overrides you wish to provide in one file. For example, if you want to provide an override with Docker credentials and a different source for EPEL on a CentOS7 machine, you can create a file like this:
 
 ```yaml
```

Comment:
```suggestion
```bash
```

The below changes turn this text snippet into a shell command.

Reply:
true true  - but - if you keep the yaml code bracket, the colors look cool in the UI ;)

Reply:
Yeah I think we've talked about this before and we like the coloring, right? Still copies well?